aten/src/ATen/native/xnnpack/RegisterOpContextClass.cpp

```diff
@@ -15,6 +15,7 @@ using internal::linear::createLinearClampPrePackOpContext;
 using internal::convolution2d::createConv2dClampPrePackOpContext;
```

Comment:
Looks like no change to this file.  Want to revert it?

Reply:
Will do.